frontend/server/src/CreateUserParams.php

```diff
@@ -57,7 +57,19 @@ class CreateUserParams {
     public $recaptcha = null;
 
     /**
-     * @param array{username?: string, name?: string, email?: string, password?: string, scholar_degree?: string, is_private?: string, gender?: string, recaptcha?: string} $params
+     * @readonly
+     * @var int
+     */
+    public $birthDate;
+
+     /**
+     * @readonly
+     * @var null|string
+     */
+    public $parentEmail = null;
+
+      /**
+     * @param array{username?: string, name?: string, email?: string, password?: string, scholar_degree?: string, is_private?: string, gender?: string, recaptcha?: string, birth_date: string, parent_email?: string} $params
```

Comment:
The birth date param should be an integer instead of a string
```suggestion
     * @param array{username?: string, name?: string, email?: string, password?: string, scholar_degree?: string, is_private?: string, gender?: string, recaptcha?: string, birth_date?: int, parent_email?: string} $params
```